docs/basic-features/font-optimization.md

```diff
@@ -26,49 +26,82 @@ Automatically self-host any Google Font. Fonts are included in the deployment an
 
 Import the font you would like to use from `@next/font/google` as a function. We recommend using [**variable fonts**](https://fonts.google.com/variablefonts) for the best performance and flexibility.
 
-```jsx
-// app/layout.tsx
-import { Inter } from '@next/font/google'
+To use the font in all your pages, add it to `_app.js` or `_app.tsx` under `/pages` as shown below:
+
+```js:pages/_app.js
+import { Inter } from '@next/font/google';
 
 // If loading a variable font, you don't need to specify the font weight
 const inter = Inter()
 
-export default function RootLayout({
-  children,
-}: {
-  children: React.ReactNode,
-}) {
+export default function MyApp({ Component, pageProps }) {
   return (
-    <html lang="en" className={inter.className}>
-      <body>{children}</body>
-    </html>
+    <main className={inter.className}>
+      <Component {...pageProps} />
+    </main>
   )
 }
 ```
 
 If you can't use a variable font, you will **need to specify a weight**:
 
-```jsx
-// app/layout.tsx
-import { Roboto } from '@next/font/google'
+```js:pages/_app.js
+import { Roboto } from '@next/font/google';
 
 const roboto = Roboto({
   weight: '400',
 })
 
-export default function RootLayout({
-  children,
-}: {
-  children: React.ReactNode,
-}) {
+export default function MyApp({ Component, pageProps }) {
+  return (
+    <main className={roboto.className}>
+      <Component {...pageProps} />
+    </main>
+  )
+}
+```
+
+#### Apply the font in `<head>`
+
+You can also use the font without a wrapper and `className` by injecting it inside the `<head>` as follows:
+
+```js:pages/_app.js
+import { Inter } from '@next/font/google';
+
+const inter = Inter();
+
+export default function MyApp({ Component, pageProps }) {
   return (
-    <html lang="en" className={roboto.className}>
-      <body>{children}</body>
-    </html>
+    <>
+      <style jsx global>{`
+        html {
+          font-family: ${inter.style.fontFamily};
+        }
+      `}</style>
+      <Component {...pageProps} />
+    </>
   )
 }
 ```
 
+#### Single page usage
+
+To use the font on a single page, add it to the specific page as shown below:
+
+```js:pages/index.js
```

Comment:
Should the example use the font in a component that the page renders, to make it clear that you don't have to call the font function in the actual page for it to work? What do you think?
```js:pages/index.js
import Header from "../components/header.js"

export default function Home() {
  return (
    <>
        <Header>Test</Header>
        <p>Hello World</p>
    </>
  );
}
```

```js:components/header.js
import { inter } from "../styles/fonts.js"

export default function Header({ children }) {
  return <h1 className={inter.className}>{children}</h1>;
}
```